src/Symfony/Component/Ldap/Adapter/EntryManagerInterface.php

```diff
@@ -40,4 +40,13 @@ public function update(Entry $entry);
      * @param Entry $entry
      */
     public function remove(Entry $entry);
+
+    /**
+     * Renames an entry on the Ldap server.
+     *
+     * @param Entry  $entry
+     * @param string $newRdn
+     * @param bool   $removeOldRdn
+     */
+    public function rename(Entry $entry, $newRdn, $removeOldRdn = true);
```

Comment:
Do we still consider the API of the LDAP component to be internal and subject to change? Otherwise, this change is a BC break.